scipy/stats/_morestats.py

```diff
@@ -900,10 +900,12 @@ def boxcox_llf(lmb, data):
 
     .. math::
 
-        llf = (\lambda - 1) \sum_i(\log(x_i)) -
-              N/2 \log(\sum_i (y_i - \bar{y})^2 / N),
+        llf = (\lambda - 1) \sum_i^N(\log(x_i)) -
```

Comment:
In the literature, is there a common symbol instead of $llf$? I'd avoid making it look like $fl^2$. $\text{LLF}$ or something?

Reply:
Anything with precedent other than $l^2f$ is fine with me.

Reply:
I looked up several books. :math:`l` seems quite common for the log likelihood (e.g. Yeo & Johnson).